src/CoreBundle/Bridge/Chameleon/Migration/Script/update-1549541799.inc.php

```diff
@@ -338,3 +340,181 @@
         TCMSLogChange::INFO_MESSAGE_LEVEL_WARNING
     );
 }
+
+/*
+ * Add remaining tables to "other" category, excluding all standard tables that are not intended to be accessed directly.
+ * This is intended as a fallback for project-specific and third-party bundle so that menu items do not disappear
+ * completely after migration.
+ */
+
+$excludeList = [
```

Comment:
This is only to prevent "disappear completely". All will show up in "Other"?

(They will be consciously added to specific categories in customer projects. Will they move now?)